bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/session-channel-types.xml

```diff
@@ -14,10 +14,18 @@
 	<channel-type id="session-energy-channel">
 		<item-type>String</item-type>
 		<label>Charged Energy in Session</label>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="session-issue-channel">
 		<item-type>String</item-type>
 		<label>Issues during Session</label>
+		<tags>
+			<tag>Control</tag>
```

Comment:
```suggestion
			<tag>Status</tag>
```

`Channel` is again not appropriate. I start thinking we need something like `Message`.

Reply:
'Info' ??

Reply:
Removed